contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -85,20 +99,23 @@ export default {
     },
   },
   actions: {
+    async saveSession(context, currentUser) {
+      await Session.put({
+        ...currentUser,
+        CURRENT_USER,
+      });
+      context.commit('ADD_SESSION', currentUser);
+    },
     login(context, credentials) {
       return client.post(window.Urls.login(), credentials);
     },
     logout(context) {
       return client.get(window.Urls.logout()).then(() => {
-        context.commit('SET_CURRENT_USER', {});
-        localStorage['loggedOut'] = true;
-        window.location = '/';
+        context.commit('REMOVE_SESSION');
```

Comment:
Is there a benefit to not clearing the indexedDB explicitly here when we know a logout has occurred? Combined with the suggestion I made above for the the conditional, this should limit the number of times we end up clearing the database.

Reply:
I think that I've done it this way because of a different set of conditions in `startApp` where logout was handled. I didn't have a particular reason for that, though. Your suggestion makes sense. It saves one reset call :+1: